### DISCUSSION TITLE
Super Slow on 4K screen

### DISCUSSION THREAD

**Discussion by a-x-** [webpack-contrib/webpack-bundle-analyzer#581] [Q&A]
#### Issue description
...

#### Technical info
* Webpack Bundle Analyzer version: 3.6.9
* Webpack version: 5.0.0-beta.12
* Node.js version: 12.13.1
* npm,yarn version: 6.13.6, 1.13.0
* OS: macOS Mojave

#### Debug info
**How do you use this module?:** as plugin

**what options were provided?:**  `new BundleAnalyzerPlugin({ })`

**What other Webpack plugins were used?:** does not matter

**Comment by valscion** [comment]
I also have a 4K screen and it does not seem to lag for me. Please share more details, for example a video of the UI lagging. Otherwise we can't debug this at all

**Comment by a-x-** [comment]
ok, 

I also see it lagging on my external 3840 x 1600 (3.5k?) monitor, but it lagging for a minute and then working fine

On 4k tv screen it lagging constantly 

### System
- MacBook Pro (15-inch, 2016)
- Radeon Pro 455 2 GB graphics
- 2,7 GHz Intel Core i7
- 16 GB 2133 MHz LPDDR3
- Safari

### Project
~4 MiB minified, no gzipped

<img width="1920" alt="Screenshot 2020-01-26 at 15 20 01" src="https://user-images.githubusercontent.com/6201068/73256615-47661a00-41d3-11ea-9cd6-60af85fa5865.png">


----

It's hard to record any meaningful, because nothing happen when I move cursor or trying to zoom in/out

Also, safari reloads page sometimes

![Screenshot 2020-01-28 at 13 39 21](https://user-images.githubusercontent.com/6201068/73256769-9a3fd180-41d3-11ea-835b-2eabdcb7754d.png)

I'll try google chrome on 4k earlier

**Comment by a-x-** [comment]
[stats.json.zip](https://github.com/webpack-contrib/webpack-bundle-analyzer/files/4121936/stats.json.zip)

**Comment by valscion** [comment]
Thanks for following up! I'll see if I can reproduce this when I've got a time slot to debug this further :relaxed:
